content/docs/developer/anteaterapi/migrating.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Migrating from PeterPortal API
+---
```

Comment:
Should add a description. Something along the lines of, "Anteater API is our new API for UCI related data, combining years of knowledge from PeterPortal API. Migrate before the legacy API is sunset."